fs/io_zenfs.cc

```diff
@@ -182,6 +177,16 @@ Status ZoneFile::DecodeFrom(Slice* input) {
       case kIsSparse:
         is_sparse_ = true;
         break;
+      case kLinkedFilename:
+        if (!GetLengthPrefixedSlice(input, &slice))
+          return Status::Corruption("ZoneFile", "LinkFilename missing");
+
+        if (slice.ToString().length() == 0)
+          return Status::Corruption("ZoneFile", "Zero length Linkfilename");
+
+        linkfiles_.push_back(slice.ToString());
+        filename_ = linkfiles_[0];
```

Comment:
I think we might as well drop filename_ and have GetFilename return linkfiles_[0]

Reply:
Or drop GetFilename..

Reply:
> I think we might as well drop filename_ and have GetFilename return linkfiles_[0]

This is better.